README-pl.md

```diff
@@ -53,11 +60,16 @@ Upewnij się, że zainstalowałeś odpowiednią wersję Hugo "extended", określ
 
 Aby uruchomić i przetestować serwis lokalnie, wykonaj:
 
-```bash
-# install dependencies
-npm ci
-make serve
-```
+- macOS i Linux
+  ```bash
```

Comment:
since few years bash is no longer default shell under Mac OS X

Reply:
See my comment above.